src/store/flow/block.ts

```diff
@@ -81,14 +81,21 @@ export const mutations: MutationTree<IFlowsState> = {
     Vue.delete(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!, key)
   },
   block_updateConfigByPath(state, {blockId, path, value}: {blockId: string, path: string, value: object | string}) {
-    // todo: this might still break reactivity
-    // Make nested assignment reactive
-    setWith(
-      findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!,
-      path,
-      value,
-      (nestedValue, key, nestedObject) => Vue.set(nestedObject, key, nestedValue),
-    )
+    const base = findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config
```

Comment:
thanks for trying to address the TODO, wonder if this approach `cloneDeep` would be more clearer ? https://bitbucket.org/voto/voto5/src/55efeb5596050fba0c9c06c0333f64d7ecef38a9/resources/js/trees/stores/trees.js#lines-198

Reply:
@jcbashdown , fyi: To answer your question here-above, another thing that I flagged in the other [PR](https://www.youtube.com/watch?v=ihrUIPfvTh8) is that we need to consider IResourceValue interface as well.